modules/serverless-creating-broker-admin.adoc

```diff
@@ -28,10 +28,11 @@ $ oc label namespace <namespace> knative-eventing-injection=enabled
 
 .Verification steps
 
-* You can verify that the broker has been created successfully by entering the following command:
+* Verify that the broker has been created successfully:
 +
 
 [source,terminal]
 ----
-$ oc -n <namespace> get broker <broker-name>
+$ oc -n <namespace> get broker <broker_name>
 ----
+// Add example output?
```

Comment:
@rhuss when you have a sec can you give me a quick example output for this so I don't have to spin up a whole cluster just to get it? :smile: